cloud_dashboard/src/service/utility.ts

```diff
@@ -66,6 +66,58 @@ const roundNumber = (value: number, digit: number) => {
   return temp;
 };
 
+const convertKeyValueCrLfData = (data: string) => {
+  const temp = data.split('\n');
+  const CONVERT_KEY_VALUE_CRLF_DICT: Record<string, string> = {
+    'on_demand_hourly: ': 'On-demand Hourly ($): ',
+    'on_demand_daily: ': 'On-demand Daily ($): ',
+    'on_demand_monthly: ': 'On-demand Monthly ($): ',
+    'on_demand_yearly: ': 'On-demand Yearly ($): ',
+    'ri_one_year: ': 'RI 1 Year ($): ',
+    'ri_three_year: ': 'RI 3 Year ($): ',
+    'cpu: ': 'CPU (Usage): ',
+    'memory: !!float ': 'Memory (Usage): ',
+    'memory: ': 'Memory (Usage): ',
+    'pod_allocation: ': 'Pods (Allocation): ',
+    'cpu_capacity: !!float ': 'CPU (Capacity): ',
+    'memory_capacity: !!float ': 'Memory (Capacity): ',
+    'pod_capacity: ': 'Pods (Capacity): ',
+  };
+  const temp2: string[] = [];
```

Comment:
@RyoYamashita765 This can be (more meaningful variable name):

```suggestion
  const results: string[] = [];
```